addons/sale_management/views/sale_order_template_views.xml

```diff
@@ -32,6 +32,7 @@
                     <group>
                         <group>
                             <field name="active" invisible="1"/>
+                            <field name="so_prefix"/>
                            <label for="number_of_days" string="Quotation expires after"/>
                            <div id="number_of_days">
```

Comment:
There seems to be an indentation difference here, do you mind fixing it in the same commit ? :)